config/webpack.config.js

```diff
@@ -21,6 +21,8 @@ const {
   getWebpackEntryPoints,
 } = require('./manifest-helpers');
 
+const vaMedaliaStylesFilename = 'va-medallia-styles';
```

Comment:
```suggestion
const vaMedalliaStylesFilename = 'va-medallia-styles';
```

Reply:
🤦‍♂️  I made that change locally earlier today, but forgot to push it. 

0207819b6edddcb09f8f52f2b5f2902d7f9c15e1